src/Weaviate.Client/AliasClient.cs

```diff
@@ -0,0 +1,81 @@
+using Weaviate.Client.Models;
+
+namespace Weaviate.Client;
+
+/// <summary>
+/// Client for managing collection aliases at the Weaviate instance level.
+/// </summary>
+public class AliasClient
+{
+    private readonly WeaviateClient _client;
+
+    internal AliasClient(WeaviateClient client)
+    {
+        _client = client ?? throw new ArgumentNullException(nameof(client));
+    }
+
+    /// <summary>
+    /// Get an alias by name
+    /// </summary>
+    /// <param name="aliasName">The name of the alias to retrieve</param>
+    /// <returns>The alias with its target collection</returns>
+    public async Task<Alias> Get(string aliasName)
```

Comment:
is throwing an exception here c#-y? Alternative would be to return an optional Alias in case of a 404